### ISSUE TITLE
Empty Spacer Column false positive on img tag

### ISSUE THREAD

**Issue by mbarlock** [twbs/bootlint#184]
image tags are mistakenly marked as an empty spacer column 

```
<img class="col-md-4 hidden-sm hidden-xs img-responsive" src="www.url.com/to/img" alt="alt-text">
```

results in this error

```
W009 Using empty spacer columns isn't necessary with Bootstrap's grid. So instead of having an empty grid column with class="col-md-4" , just add class="col-md-offset-4" to the next grid column.
```

**Comment by cvrebert** [comment]
@mdo Can you clarify what's valid usage here?

**Comment by mdo** [comment]
You can place column classes on anything, but some elements work better (e.g., `<div>`s). I'd prefer the example you mentioned @cvrebert.

**Comment by cvrebert** [comment]
@hnrch02 I'm inclined to close this as WontFix. What's your opinion?

**Comment by hnrch02** [comment]
I'd say we exempt self closing tags from W009? Technically, `img.col-md-4` is valid usage as @mdo mentioned, so I don't think we shouldn't warn about it.

**Comment by hnrch02** [comment]
@cvrebert We could also make a new warning though just for this case, your call.

**Comment by cvrebert** [comment]
> I'd say we exempt self closing tags from W009?

Sounds reasonable. Your call as maintainer.

> We could also make a new warning though just for this case

Doesn't seem worth it, but I wouldn't actively oppose it.
